api/synapse-api-rest-imperative/pom.xml

```diff
@@ -36,10 +36,10 @@
             <groupId>io.americanexpress.synapse</groupId>
             <artifactId>synapse-framework-exception</artifactId>
         </dependency>
-        <dependency>
-            <groupId>io.americanexpress.synapse</groupId>
-            <artifactId>synapse-framework-api-docs</artifactId>
-        </dependency>
+<!--        <dependency>-->
```

Comment:
Was this commented code on purpose?

Reply:
removed. yes, I had left it to revisit, but we have task to revisit later.